sdk/template/ci.yml

```diff
@@ -25,18 +26,9 @@ pr:
   paths:
     include:
     - sdk/template/
-#if (false)
```

Comment:
Did you intend to remove these?

Reply:
updated

Reply:
Looks like this if condition is still removed I suspect we still want these paths here.

Reply:
Hi @heaths Shall we still add this if condition here in ci.yml? Thanks

Reply:
I'm going to say we need to keep this removed section so that our template will still trigger to validate things under those paths. I'm not sure why this ended up getting removed but I think we should definitely add it back before merging this change.

Reply:
@chunyu3 can you please add this back? We currently do not have pipeline runs for various engsys related PRs that rely on the template tests as our primary CI gate.